spyder/plugins/configdialog.py

```diff
@@ -317,7 +317,12 @@ def load_from_conf(self):
             radiobutton.toggled.connect(lambda _foo, opt=option:
                                         self.has_been_modified(opt))
         for lineedit, (option, default) in list(self.lineedits.items()):
-            lineedit.setText(self.get_option(option, default))
+            option_value = self.get_option(option, default)
+            if isinstance(option_value, tuple):
+                option_value = ",".join(str(e) for e in option_value)

```

Comment:
`str` returns bytes for Python 2 and Unicode for Python 3. We should use only format here.

So I think we should change this call to `str` (and the one below) to `to_text_string`